### PR TITLE
Created draft of one pager for Library Admin

### PR DESCRIPTION
**Pull request by arieldeardorff**

### CODE REVIEW

File: proposals/benefits_of_lc.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+Using the pedagogical practices of live coding, pair programming, discussion and exercises, [Library Carpentry workshops](https://librarycarpentry.org/lessons/) create a safe and collaborative space for important concepts in computing and data, including data manipulation and organization, text and data mining, and workflow versioning and management. Not only do these skills help librarians create reproducible workflows and reliable operations for data-centric tasks, they give librarians a common language with researchers that can lead to a better mutual understanding of data issues and it paves the way to greater collaboration between the library and their research community.
+
+Since 2015, Library Carpentry has held [over 75 workshops worldwide](https://librarycarpentry.org/past_workshops/) and three global sprints involving hundreds of contributors to improve the lesson materials. Librarians who have attended workshops have reported improved abilities in automating curation workflows, communicating more effectively with researchers, collaborating on projects, and generating reports more efficiently, to name a few. (add more examples from Elizabeth here).
```
Comment (libcce) [inline proposals/benefits_of_lc.md:10]: The workshop count is not accurate. It was 142 total workshops in November which you can see here https://github.com/LibraryCarpentry/librarycarpentry.github.io/blob/06cb6f82286006c8f1b860c76f9554dcfec6cbe3/_data/workshops.yml @maneesha can this be fixed?

File: proposals/benefits_of_lc.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+Since 2015, Library Carpentry has held [over 75 workshops worldwide](https://librarycarpentry.org/past_workshops/) and three global sprints involving hundreds of contributors to improve the lesson materials. Librarians who have attended workshops have reported improved abilities in automating curation workflows, communicating more effectively with researchers, collaborating on projects, and generating reports more efficiently, to name a few. (add more examples from Elizabeth here). 
+
+## Why Become a Member?
+Library Carpentry offers a lightweight opportunity to provide ongoing professional development to library teams and low-cost pathways for organizations or networks to get started. By [becoming a member of the Carpentries](https://carpentries.org/membership/), Libraries are able to host workshops, get priority access to instructor training, and contribute to the direction of the organization. Membership fees start at $7,500 per year for a silver level membership that includes 4 coordinated workshops and the 6 spots in instructor training.
```
Comment (libcce) [inline proposals/benefits_of_lc.md:13]: Maybe refer to this as "certified instructors" and developing a cohort, see NESCLIC example https://nesclic.github.io/

**Comment by libcce** [comment]
One other place we can pull from https://librarycarpentry.org/lc-overview/02-intro-to-library-carpentry/index.html

File: proposals/benefits_of_lc.md (hunk 1 above)
Comment (arieldeardorff) [reply proposals/benefits_of_lc.md:10]: I will update the number and remove the link for now

File: proposals/benefits_of_lc.md (hunk 2 above)
Comment (arieldeardorff) [reply proposals/benefits_of_lc.md:13]: Sounds good!

**Comment by arieldeardorff** [comment]
I think I will move this page to a google doc for now as that will be way easier to get feedback/edits. We can upload the final version here in GitHub

**Pull request closed without merging**